native/explorer/src/datatypes.rs

```diff
@@ -65,24 +65,32 @@ pub struct ExDataFrame {
     pub resource: ResourceArc<ExDataFrameRef>,
 }
 
+impl std::panic::RefUnwindSafe for ExDataFrame {}
```

Comment:
This is required because of the change introduced in this PR: https://github.com/rusterlium/rustler/pull/656